debian/changelog

```diff
@@ -0,0 +1,5 @@
+alire (1.2.0) unstable; urgency=low
```

Comment:
Debian policy [5.6.12.2. Special version conventions](https://www.debian.org/doc/debian-policy/ch-controlfields.html#special-version-conventions) says:
> The absence of debian_revision, and therefore of a hyphen in the version number, indicates that the package is native.

I believe this should be `1.2.0-1` to indicate a non-native package.